lms/djangoapps/learner_dashboard/tests/test_programs.py

```diff
@@ -0,0 +1,66 @@
+"""
+Tests for viewing the programs enrolled by a learner.
+"""
+import datetime
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from opaque_keys.edx import locator
+import unittest
+import ddt
+
+from student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+from student.models import CourseEnrollment
+from common.test.utils import XssTestMixin
+from mixins import LearnerDashboardConfigMixin
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+@ddt.ddt
+class TestProgramListing(ModuleStoreTestCase, XssTestMixin, LearnerDashboardConfigMixin):
```

Comment:
Since you're creating but not otherwise modifying course data in these tests, I'd encourage you to use [SharedModulestoreTestCase](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/tests/django_utils.py#L230).